src/main/java/com/thullo/data/model/BoardVisibility.java

```diff
@@ -0,0 +1,25 @@
+package com.thullo.data.model;
+
+import com.thullo.web.exception.BadRequestException;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public enum BoardVisibility {
+    PRIVATE,
+    PUBLIC;
+
+
+    public static BoardVisibility getBoardVisibility(String input) throws BadRequestException {
+        List<BoardVisibility> visiblityList = Arrays.asList(
```

Comment:
instead of creating an array you can use `BoardVisibility.values`, also when you're comparing the input use `equalsIgnoreCase`